web/app/js/components/ServiceMesh.jsx

```diff
@@ -32,9 +32,15 @@ const componentNames = {
   "proxy-api":    "Controller proxy-api",
   "public-api":   "Controller public-api",
   "tap":          "Controller tap",
-  "telemetry":    "Controller telemetry",
+  "telemetry":    "Controller Telemetry",
```

Comment:
If we're going to make this capitalization change, I'd like to change the rest of the names in this map as well, as follows:

"Controller Destination"
"Controller Proxy API"
"Controller Public API"
"Controller Tap"
"Controller Telemetry"